white-test-app-backend/src/main/java/pl/lodz/p/white/whitetestapp/model/Browser.java

```diff
@@ -0,0 +1,5 @@
+package pl.lodz.p.white.whitetestapp.model;
+
+public enum Browser {
+    C, F, I;
```

Comment:
Wydaje się to ograniczone rozwiązanie, ale  widzę, że zgodne z specyfikacją zadania